assets/xml/objects/object_boss02.xml

```diff
@@ -1,99 +1,125 @@
-        <Limb Name="object_boss02_Standardlimb_013920" Type="Standard" EnumName="OBJECT_BOSS02_2_LIMB_17" Offset="0x13920" />
-        <Skeleton Name="object_boss02_Skel_013988" Type="Flex" LimbType="Standard" LimbNone="OBJECT_BOSS02_2_LIMB_NONE" LimbMax="OBJECT_BOSS02_2_LIMB_MAX" EnumName="object_boss02_2_Limbs" Offset="0x13988" />
+        <!-- DisplayLists and Textures for the dust that appears wherever Twinmold contacts the ground -->
+        <Texture Name="gTwinmoldDustMaskTex" OutName="dust_mask" Format="i4" Width="32" Height="32" Offset="0x0" />
+        <DList Name="gTwinmoldDustMaterialDL" Offset="0x230" />
+        <DList Name="gTwinmoldDustModelDL" Offset="0x2E0" />
+
+        <!-- Twinmold Textures -->
+        <Texture Name="gTwinmoldSkinTLUT" OutName="twinmold_skin_tlut" Format="rgba16" Width="4" Height="4" Offset="0x300" />
+        <Texture Name="gTwinmoldSnoutTLUT" OutName="twinmold_snout_tlut" Format="rgba16" Width="4" Height="4" Offset="0x320" />
+        <Texture Name="gTwinmoldVeinsTLUT" OutName="twinmold_veins_tlut" Format="rgba16" Width="4" Height="4" Offset="0x340" />
+        <Texture Name="gTwinmoldUndersideTLUT" OutName="twinmold_underside_tlut" Format="rgba16" Width="4" Height="4" Offset="0x360" />
+        <Texture Name="gTwinmoldSpikeAndMandibleTLUT" OutName="twinmold_spike_and_mandible_tlut" Format="rgba16" Width="4" Height="4" Offset="0x380" />
+        <Texture Name="gTwinmoldRedSkinTex" OutName="twinmold_red_skin" Format="ci4" Width="64" Height="64" Offset="0x3A0" TlutOffset="0x300" />
+        <Texture Name="gTwinmoldEyeTex" OutName="twinmold_eye" Format="rgba16" Width="32" Height="32" Offset="0xBA0" />
+        <Texture Name="gTwinmoldSnoutTex" OutName="twinmold_snout" Format="ci4" Width="64" Height="64" Offset="0x13A0" />
+        <Texture Name="gTwinmoldVeinsTex" OutName="twinmold_veins" Format="ci4" Width="64" Height="64" Offset="0x1BA0" />
+        <Texture Name="gTwinmoldTeethTex" OutName="twinmold_teeth" Format="rgba16" Width="16" Height="32" Offset="0x23A0" />
+        <Texture Name="gTwinmoldUndersideTex" OutName="twinmold_underside" Format="ci4" Width="64" Height="64" Offset="0x27A0" />
+        <Texture Name="gTwinmoldSpikeAndMandibleTex" OutName="twinmold_spike_and_mandible" Format="ci4" Width="64" Height="64" Offset="0x2FA0" />
+        <Texture Name="gTwinmoldAntennaTex" OutName="twinmold_antenna" Format="rgba16" Width="16" Height="16" Offset="0x37A0" />
+        <Texture Name="gTwinmoldTailGemTex" OutName="twinmold_tail_gem" Format="rgba16" Width="32" Height="32" Offset="0x39A0" />
+        <Texture Name="gTwinmoldBlueSkinTex" OutName="twinmold_blue_skin" Format="ci4" Width="64" Height="64" Offset="0x41A0" TlutOffset="0x300"/>
+
+        <!-- Unused, completely motionless Twinmold Head animation -->
+        <Animation Name="gTwinmoldHeadStationaryAnim" Offset="0x4A1C" />
```

Comment:
Hmm, is this not used when a Twinmold dies and the head falls to the ground?

Reply:
That may have been its intention, but that's not what it's used for in the final game (it's not used for anything, mind you). I think what happens is they just stop calling `SkelAnime_Update` once it dies.

Reply:
Just looked at it in-game and I'm almost certain I'm right. The head's animation "freezes" in whatever state it's in when it dies, rather than going back to the "default" state shown in this unused animation.